### PR TITLE
ignore package-lock.json

### PR DESCRIPTION
**Pull request by yoshuawuyts**
We're not committing it, and I was getting a lil annoyed by it popping up all the time. This should fix that minor grievance. Thanks!

### CODE REVIEW

**Comment by goto-bus-stop** [comment]
Good, thanks :P just earlier today I learned about project specific `.npmrc`: https://github.com/browserify/browserify/pull/1799#pullrequestreview-88738043
would you mind adding one of those too?

**Comment by yoshuawuyts** [comment]
@goto-bus-stop done!

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (YerkoPalma) [inline .npmignore]: I though this change was for a `.npmrc` file or is this the same?

Comment (YerkoPalma) [reply .npmignore]: cool :+1:

Comment (yoshuawuyts) [reply .npmignore]: lol, sorry :P

**Pull request merged**